resources/g2/sprites.json

```diff
@@ -18341,13 +18341,13 @@
         "palette": "green"
     },
     {
-        "path": "track/bm/brake_horizontal.png",
+        "path": "track/bm_invert/brake_horizontal.png",
         "x_offset": -32,
         "y_offset": 2,
         "palette": "green"
     },
     {
-        "path": "track/bm/brake_vertical.png",
+        "path": "track/bm_invert/brake_vertical.png",
```

Comment:
why this change?

Reply:
There's already a folder called bm_invert, and since this is a B&M Invert track I am keeping them together.

Reply:
oh right I forgot that was added